domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java

```diff
@@ -81,7 +81,8 @@ public void deleteTenant(Map<String, String> headers,
 
             String sqlFile = null;
             try {
-              SchemaMaker sMaker = new SchemaMaker(tenantId, PostgresClient.getModuleName(), TenantOperation.DELETE, null, PomReader.INSTANCE.getRmbVersion());
+              SchemaMaker sMaker = new SchemaMaker(tenantId, PostgresClient.getModuleName(),
+                  TenantOperation.DELETE, null, RmbVersion.getRmbVersion());
```

Comment:
Not sure why set module previous version to null and module module version to RMB version here.

Reply:
Good catch. We neither use nor need any version on delete. I've replaced the RMB version by null. For code readability we should not pass a value for a parameter that is not used.